pkg/labels/labels.go

```diff
@@ -101,6 +101,9 @@ const (
 	// (like "nerdctl/default-network=true" or "nerdctl/default-network=false")
 	NerdctlDefaultNetwork = Prefix + "default-network"
 
+	// LogConfig defines the loggin configuration passed to the container
```

Comment:
typo: loggin

Reply:
The `LogURI` label can be now removed?

Reply:
> The `LogURI` label can be now removed?

Not sure about this, I believe logUri is used in log persistence between container restarts, will dig deeper and see if this is feasible

Reply:
Not sure about removing the label, seem like it is being used in the attach command, that change should be in a separate PR I think.